### ISSUE TITLE
Accept header on importing OpenAPI 3 spec

### ISSUE THREAD

**Issue by drjonnicholson** [postmanlabs/postman-app-support#5933]
**Is your feature request related to a problem? Please describe.**

When importing an OpenAPI 3 specification no accept header is set in the postman collection. In our case (good or bad) we specify the api version in the media type in the accept header and we need to manually override this after importing the specification for requests to work.

That is, an accept header of */* doesn’t work for us as the API I’m documenting will throw a 400 error if a valid media type is not presented in the accept header.

**Describe the solution you'd like**

Have the accept header set, by default to the first content type of the 200 response (and description set listing any others supported if specified?)

**Describe alternatives you've considered**

Considered manually adding an accept header explicitly in the spec. But this is noted as ignored in the OA3 definition, and causes issues in our code generation tool chain.

An appropriate default accept header value should, ideally, be inferred from the specified content types of the responses.

Would be happy if a vender extension were needed to specify what the accept header would default to. For example, some code generation targets use x-accept for this purpose. I wonder if something like this already exists but isn’t documented?

Basically I want to import our spec into postman and it “just work” - the missing accept header appears to be the only barrier to us achieving this.

**Additional context**

None at this time
